<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>551</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Swabian</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="APD0_label">
      <property name="text">
       <string>APD 0 Channel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="channel0_label">
      <property name="text">
       <string>Channel 0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QCheckBox" name="continuous_checkBox">
      <property name="text">
       <string>Continuous</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="ch0_label">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="APD0_select">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>2</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="APD1_select_label">
      <property name="text">
       <string>APD 1 Channel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="counter_bin_label">
      <property name="text">
       <string># of Counter Bins</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="APD1_select">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>3</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QComboBox" name="trigger_select">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>1</string>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QDoubleSpinBox" name="counter_binwidth_doubleSpinBox">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000000000000.000000000000000</double>
      </property>
      <property name="value">
       <double>100000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="trigger_label">
      <property name="text">
       <string>Laser Trigger Channel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QDoubleSpinBox" name="Tacq_doubleSpinBox">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>100000000000.000000000000000</double>
      </property>
      <property name="value">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="counter_binwidth_label">
      <property name="text">
       <string>Counter Bin Width (ps)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="Tacq_bin_label">
      <property name="text">
       <string>Tacq (seconds)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="channel1_label">
      <property name="text">
       <string>Channel 1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="ch1_label">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QComboBox" name="BinWidth_Unit_Select">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>ps</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="maxVisibleItems">
       <number>5</number>
      </property>
      <item>
       <property name="text">
        <string>ps</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ns</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Âµs</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>s</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="g2_numbins_doubleSpinBox">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>100000000000000000.000000000000000</double>
      </property>
      <property name="value">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="g2_binwidth_label">
      <property name="text">
       <string>g2(t) Bin Width</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QDoubleSpinBox" name="g2_binwidth_doubleSpinBox">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>100000000000000000.000000000000000</double>
      </property>
      <property name="value">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="g2_bin_label">
      <property name="text">
       <string># of g2(t) Bins</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="counter_numbins_doubleSpinBox">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>100000000000.000000000000000</double>
      </property>
      <property name="value">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Run</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="save_data_pushButton">
      <property name="text">
       <string>Save data</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="clear_plot_pushButton">
      <property name="text">
       <string>Clear Plot</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="interrupt_pushButton">
      <property name="text">
       <string>Interrupt</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="start_pushButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="plot_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>751</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>Plot</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
